components/service/experiments/src/main/java/mozilla/components/service/experiments/ExperimentsUpdater.kt

```diff
@@ -88,3 +133,21 @@ internal class ExperimentsUpdater(
         private const val EXPERIMENTS_COLLECTION_NAME = "mobile-experiments"
     }
 }
+
+/**
+ * This class represents a [Worker] object that periodically updates the experiments.
+ * @param updater used so that the updater can be injected from tests.
+ */
+internal class ExperimentsUpdaterWorker(
```

Comment:
Is it a typical style to have this at the bottom?

From the desktop code and other projects i'm used to having helpers & utils before the main class.
That helps with actually discovering them, i wouldn't think of looking for other classes after this files main class.

Unless there is a good reason, let's maybe move this one up to before `class ExperimentsUpdater`?

Reply:
It's fairly common in mobile to have extensions and stuff at the end of the file, but I see your point.  Maybe it would be _best_ if I just put it in it's very own file?

Reply:
Nevermind, I remember now why I did it in the same file...  due to visibility warnings/issues of how Kotlin treats `private` as rather `file private` than `class private`.  The worker needed access to call the updateExperiments() function.   I'll just move it to the top as you suggested.